### ISSUE TITLE
[Feature Request] Disable crit attacks

### ISSUE THREAD

**Issue by Flemmli97** [Shadows-of-Fire/Apotheosis#711]
It would be nice if there is a way to disable crit hits. Either via config or maybe when sneaking. Since critting means no sweeping edge which is sometimes undesirable. furthermore the base crit value is 1.5 but the book tells you it is 1 so not sure which one is unintended

**Comment by Shadows-of-Fire** [comment]
You can disable the crit affixes via datapack if you so desire, or you can just, you know, not use weapons with crit chance on them.

The book value is wrong there, crit has undergone many recent rebalancings.

**Comment by Flemmli97** [comment]
well since the base value is 1.5 it means you still have a 50% chance to crit regardless of weapon so you cannot disable it

**Comment by Shadows-of-Fire** [comment]
The base value of crit *damage* is 1.5, which reflects the vanilla modifier on jump crits.  Crit *chance* has a base value of 1 (which is zero)

**Comment by Flemmli97** [comment]
i think you swapped them then [https://github.com/Shadows-of-Fire/Apotheosis/blob/1.19/src/main/java/shadows/apotheosis/adventure/AdventureModule.java#L210](https://github.com/Shadows-of-Fire/Apotheosis/blob/1.19/src/main/java/shadows/apotheosis/adventure/AdventureModule.java#L210)
1.18 ver too

**Comment by Shadows-of-Fire** [comment]
That's unfortunate.  Even more so that you can't retroactively change the base value because of how attributes work.

Bleh

**Comment by James103** [comment]
Workaround (for players): Run the following two commands, which edits the base value of Crit Chance and Crit Damage to be more in line with expectations and resemble vanilla behavior.
```mcfunction
attribute @s apotheosis:crit_chance base set 1.0
attribute @s apotheosis:crit_damage base set 1.5
```

**Comment by Shadows-of-Fire** [comment]
The base values have been fixed correctly in 1.19, but I need to double-check 1.18, and perhaps publish a fix there still.

**Comment by Shadows-of-Fire** [comment]
Fixed in 5.8.0
